backend/src/main/java/com/woowacourse/gongcheck/exception/ErrorResponse.java

```diff
@@ -0,0 +1,27 @@
+package com.woowacourse.gongcheck.exception;
+
+import org.springframework.web.bind.MethodArgumentNotValidException;
+
+public class ErrorResponse {
+
+    private String message;
+
+    private ErrorResponse() {
+    }
+
+    private ErrorResponse(final String message) {
+        this.message = message;
+    }
+
+    public static ErrorResponse from(final RuntimeException e) {
+        return new ErrorResponse(e.getMessage());
+    }
+
+    public static ErrorResponse from(final MethodArgumentNotValidException e) {
+        return new ErrorResponse(e.getAllErrors().get(0).getDefaultMessage());
```

Comment:
```suggestion
        return new ErrorResponse(e.getAllErrors()
            .get(0)
            .getDefaultMessage());
```
한라인에는 하나의 기능만 바라볼 수 있게끔 해주세요 :)